app/marketing/management/commands/pull_stats.py

```diff
@@ -139,6 +139,22 @@ def profiles_ingested():
         )
 
 
+def user_actions():
+    from dashboard.models import UserAction
+
+    for action_type in UserAction.ACTION_TYPES:
+        action_type = action_type[0]
+
+        val = UserAction.objects.filter(
+            action=action_type,
+            ).count()
+
+        Stat.objects.create(
+            key='user_action_{}'.format(action_type),
```

Comment:
This will work, but you could use literal string interpolation via: `key=f'user_action_{action_type}',`
Sadly, `string.format()` is slow :( - f-strings are typically as fast as the old style: `key='user_action_%s' % action_type,`

Reply:
thanks.. i need to get in the habit of doing fstrings